ajax/libs/pickmeup/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "pickmeup",
+  "filename": "pickmeup.min.js",
+  "description": "Really simple, powerful, customizable and lightweight standalone datepicker",
+  "keywords": [
+    "datepicker",
+    "date",
+    "calendar"
+  ],
+  "version": "3.0.11",
+  "homepage": "https://cleverstyle.org/",
+  "author": {
+    "name": "Nazar Mokrynskyi",
+    "email": "nazar@mokrynskyi.com",
+    "url": "https://cleverstyle.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nazar-pc/PickMeUp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nazar-pc/PickMeUp.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "{js,css}/?(jquery.)pickmeup*.+(js|css|map)"
```

Comment:
(It's Memo not request, I will request review from Peter to make final decision)
Though the current pattern can get the files we need, I prefer a simpler one: `+(js|css)/*pickmeup*.+(js|css|map)`.